.github/workflows/dapr-perf.yml

```diff
@@ -358,9 +358,14 @@ jobs:
           script: |
             const testPayload = context.payload.client_payload;
             if (testPayload && testPayload.command == "ok-to-perf") {
+              let selectedTestsEnvVar = ""
+              if (typeof testPayload.args !== "undefined") {
```

Comment:
Yes, but It was introduced because now I'm joining the array even if the size is 0, which is OK. I don't like to rely on the `falsy` behavior of js though, but I'll keep it as you suggested.